etc/config/hook.amazon.properties

```
@@ -0,0 +1,21 @@
+
+interpreted=true
+supportsBinary=false
+supportsExecute=true
+versionFlag=--version
+needsMulti=false
+
+compilerType=hook
+compilers=&hook
+defaultCompiler=hook010
+
+group.hook.compilers=hook010
+group.hook.groupName=Hook
+group.hook.isSemVer=true
+group.hook.baseName=Hook
+group.hook.licenseName=MIT
+group.hook.licenseLink=https://github.com/fabiosvm/hook-lang/blob/main/LICENSE
+
+compiler.hook010.exe=/opt/compiler-explorer/hook-0.1.0/bin/hook
```

I decided to move this into a subdir:
```suggestion
compiler.hook010.exe=/opt/compiler-explorer/hook/hook-0.1.0/bin/hook
```

but I'll merge and fix that up

---

Thanks!